tests/test_import.py

```diff
@@ -96,3 +96,25 @@ def test_invalid_numeric_datatype_scale(here, engine, woningbouwplannen_schema,
     assert not record[0]["numeric_scale"]
     assert record[1]["data_type"] == "numeric"
     assert not record[1]["numeric_scale"]
+
+
+def test_biginteger_datatype(here, engine, woningbouwplannen_schema, dbsession):
+    """Prove that when biginter is used as datatype in schema, the datatype
+    in the database is set to datatype int(8) instead of int(4)"""
+    importer = BaseImporter(woningbouwplannen_schema, engine)
+    importer.generate_db_objects("woningbouwplan", ind_tables=True, ind_extra_index=True)
+    record = [
+        dict(r)
+        for r in engine.execute(
```

Comment:
`engine.execute()` returns a [ResultProxy](https://docs.sqlalchemy.org/en/13/core/connections.html#sqlalchemy.engine.ResultProxy). If you only expect one or only need the first row you can call `fetchone()` on the `ResultProxy.` The [RowProxy](https://docs.sqlalchemy.org/en/13/core/connections.html#sqlalchemy.engine.RowProxy) returned by `fetchone()` acts like a NamedTuple or dictionary. Meaning you don't need the list comprehension and the dict. In addition the `WHERE 1=1` is superfluous Combining everything you end up with:

```python
    results = engine.execute(
        """
        SELECT data_type, numeric_precision
        FROM information_schema.columns
        WHERE table_schema = 'public'
          AND table_name = 'woningbouwplannen_woningbouwplan'
          AND column_name = 'id_big';
    """
    )
    record = results.fetchone()
    assert record.data_type == "bigint"
    assert record.numeric_precision == 64
```
This is more succinct and expresses the intention arguably better.

Reply:
:+1: Ah that much cleaner
The reason why I use where 1=1 is for the layout. The keep all the conditions with the same way (and ....) But where 1=1 can be removed because it has no use other then that